docs/content.zh/docs/dev/datastream/execution/execution_configuration.md

```diff
@@ -45,46 +45,45 @@ var executionConfig = env.getConfig
 
-- `getMaxParallelism()` / `setMaxParallelism(int parallelism)` Set the default maximum parallelism for the job. This setting determines the maximum degree of parallelism and specifies the upper limit for dynamic scaling.
+- `getMaxParallelism()` / `setMaxParallelism(int parallelism)`。为作业设置默认的最大并行度。此设置决定最大并行度并指定动态缩放的上限。
 
-- `getNumberOfExecutionRetries()` / `setNumberOfExecutionRetries(int numberOfExecutionRetries)` Sets the number of times that failed tasks are re-executed. A value of zero effectively disables fault tolerance. A value of `-1` indicates that the system default value (as defined in the configuration) should be used. This is deprecated, use [restart strategies]({{< ref "docs/ops/state/task_failure_recovery" >}}#restart-strategies) instead.
+- `getNumberOfExecutionRetries()` / `setNumberOfExecutionRetries(int numberOfExecutionRetries)`。设置失败任务重新执行的次数。值为零会有效地禁用容错。`-1` 表示使用系统默认值（在配置中定义）。该配置已弃用，请改用[重启策略]({{< ref "docs/ops/state/task_failure_recovery" >}}#restart-strategies) 。
 
-- `getExecutionRetryDelay()` / `setExecutionRetryDelay(long executionRetryDelay)` Sets the delay in milliseconds that the system waits after a job has failed, before re-executing it. The delay starts after all tasks have been successfully stopped on the TaskManagers, and once the delay is past, the tasks are re-started. This parameter is useful to delay re-execution in order to let certain time-out related failures surface fully (like broken connections that have not fully timed out), before attempting a re-execution and immediately failing again due to the same problem. This parameter only has an effect if the number of execution re-tries is one or more. This is deprecated, use [restart strategies]({{< ref "docs/ops/state/task_failure_recovery" >}}#restart-strategies) instead.
+- `getExecutionRetryDelay()` / `setExecutionRetryDelay(long executionRetryDelay)`。设置系统在作业失败后重新执行之前等待的延迟（以毫秒为单位）。在 TaskManagers 上成功停止所有任务后，开始计算延迟，一旦延迟过去，任务会被重新启动。此参数对于延迟重新执行很有用，以便在尝试重新执行并由于相同的问题立即再次失败之前，让某些超时相关的故障完全浮出水面（例如尚未完全超时的断开连接）。此参数仅在执行重试次数为一次或多次时有效。该配置已被弃用，请改用[重启策略]({{< ref "docs/ops/state/task_failure_recovery" >}}#restart-strategies) 。
 
-- `getExecutionMode()` / `setExecutionMode()`. The default execution mode is PIPELINED. Sets the execution mode to execute the program. The execution mode defines whether data exchanges are performed in a batch or on a pipelined manner.
+- `getExecutionMode()` / `setExecutionMode()`。默认的执行模式是 PIPELINED。设置执行模式以执行程序。执行模式定义了数据交换是以批处理方式还是以流方式执行。
 
-- `enableForceKryo()` / **`disableForceKryo`**. Kryo is not forced by default. Forces the GenericTypeInformation to use the Kryo serializer for POJOs even though we could analyze them as a POJO. In some cases this might be preferable. For example, when Flink's internal serializers fail to handle a POJO properly.
+- `enableForceKryo()` / **`disableForceKryo`**。默认情况下不强制使用 Kryo。强制 GenericTypeInformation 对 POJO 使用 Kryo 序列化器，即使我们可以将它们作为 POJO 进行分析。在某些情况下，这可能更可取。例如，当 Flink 的内部序列化器无法正确处理 POJO 时。
```

Comment:
```suggestion
- `enableForceKryo()` / **`disableForceKryo`**。默认情况下不强制使用 Kryo。强制 GenericTypeInformation 对 POJO 使用 Kryo 序列化器，即使我们可以将它们作为 POJO 进行分析。在某些情况下，此配置可能被优先启用。例如，当 Flink 的内部序列化器无法正确处理 POJO 时。
```
Only a minor comment. Please let me know what's your opinion.

Reply:
I thought about it.How about the following translation ?
```txt
在某些情况下这样配置可能更可取。
```

Reply:
IMO, Free translation is better than literal translation.
Hi, @wuchong ,What do your think of it ?

Reply:
I think it would be more appropriate to translate into active sentences.
How about the following translation ?
```txt
在某些情况下，优先启用该配置可能是更可取的。
```
Or the following translation？
```txt
在某些情况下，应该优先启用该配置。
```

Reply:
I'm fine with the last one.

Reply:
Thank you for your review .
I updated it as follows:
```txt
在某些情况下，应该优先启用该配置。
```